src/structs.jl

```diff
@@ -208,6 +208,8 @@ Base.@kwdef struct Parameter
     units::Maybe{String} = nothing
     constant::Maybe{Bool} = nothing
     metaid::Maybe{String} = nothing
+    notes::Maybe{String} = nothing
+    annotation::Maybe{String} = nothing
```

Comment:
I was going to suggest to add `Parameter` to the list of structs with annotations in the tests, but somehow it was already there: https://github.com/LCSB-BioCore/SBML.jl/blob/0eb0a446b62b5072e851d61fcab1e2e0c66b8124/test/common.jl#L25-L29 :sweat_smile:

Reply:
We could actually use the same `==` definition (the one which checks for `annotations` and `notes`) for all structs, but I think it'll be slightly slower for structs without annotations.  Performance is probably not a huge concern here though.